test/controllers/ht_approval_requests_controller_test.rb

```diff
@@ -23,6 +23,7 @@ def setup
     get ht_approval_requests_url
     validator = W3CValidators::NuValidator.new
     w3c_errs = validator.validate_text(@response.body).errors
+    sleep 1
```

Comment:
This is because it's sending it to an online service because it's rate-limiting us? If so, perhaps we should create an issue for running our own copy when running tests -- e.g. should be easy to spin up https://hub.docker.com/r/validator/validator/ via docker-compose when running tests

Reply:
Yes. I didn't see a timeout until after the second commit on the PR resulting in the tests failing. I saw it happen once locally, following that (could also have been a transient, who knows). I noted that the validator gem's author mentions a `sleep 1` call in that repo's tests, so just went with that for the time being.

Reply:
:+1: